Makefile

```diff
@@ -94,7 +94,8 @@ tests:
 	go test ./...
 
 vet:
-	go vet ./...
+	go tool vet --all .
+	! go tool vet --shadow . 2>&1 | grep -vE 'declaration of err shadows declaration at (proto/text.go:535|proto/text.go:570|protoc-gen-gogo/main.go:84)'
```

Comment:
The problem is that those are false positives; see https://github.com/golang/go/issues/11843